examples/js/controls/ControlSwitcher.js

```diff
@@ -0,0 +1,57 @@
+function ControlSwitcher(fov, aspect, near, far, container, onCameraChanged) {
```

Comment:
If you would like help with your formatting, see the Wiki article [Mr.doob's-Code-Style™](https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style™).